### DISCUSSION TITLE
Export package로 별도 프로젝트에서 필요한 부분만 작업 후 XTown에 import하여 사용하는 방법

### DISCUSSION THREAD

**Discussion by AliceOfSNU** [XREAL-SNU/xTown#39] [Ideas]
지금 XTown 프로젝트가 크기도 있고 그래서 거기서 바로 작업하면 build도 느리고
컴퓨터도 종종 뻗을라 그러고 그러던데

별도 프로젝트에서 필요한 기능을 작업한 후, 
XTown에 프래팹 포함, 추가할 폴더들을 선택하여 아래와 같이 Export하여
![image](https://user-images.githubusercontent.com/86138312/148345908-f538c523-2e98-49ef-b914-5d9ab62b6d21.png)

XTown에서 import asset하면 간편하게 사용할 수 있네요
이렇게 작업하니까 예를 들어서 채팅같은거 밖에서 작업하고 UI들이랑 스크립트붙은 오브젝트들 쭉 프리팹으로 뽑아서
export하고 XTown에다가는 드래그엔 드롭만 하면 되서 좋아서 공유합니다
![image](https://user-images.githubusercontent.com/86138312/148345664-f5b9ad82-6736-4ae0-9231-a5291fd56c03.png)

**Comment by brainini** [comment]
오 되게 좋은 방법이네요!! 참고하면 좋을 듯 합니다.
